Patch is project specific, and not a candidate for drupal contribution.
diff --git a/src/Access/LatestVersionCheck.php b/src/Access/LatestVersionCheck.php
index 9bc5497..83e33ce 100644
--- a/src/Access/LatestVersionCheck.php
+++ b/src/Access/LatestVersionCheck.php
@@ -9,11 +9,12 @@ use Drupal\Core\Routing\Access\AccessInterface;
 use Drupal\Core\Routing\RouteMatchInterface;
 use Drupal\Core\Session\AccountInterface;
 use Symfony\Component\Routing\Route;
+use Drupal\content_moderation\Access\LatestRevisionCheck;

 /**
  * Allow access to the latest version tab for editors and reviewers.
  */
-class LatestVersionCheck implements AccessInterface {
+class LatestVersionCheck extends LatestRevisionCheck implements AccessInterface {

   /**
    * The content moderation latest version access service.
